product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -17,8 +17,8 @@ Postgres distribution and version support varies by cluster type.
 | --------------------- | ---------------- | ------------------------------ |
 | PostgreSQL            | 11–15            | Single node, high availability |
 | Oracle Compatible     | 11–15            | Single node, high availability |
-| Oracle Compatible     | 12–14            | Extreme high availability      |
-| PostgreSQL Compatible | 12-14            | Extreme high availability      |
+| Oracle Compatible     | 14–15            | Extreme high availability      |
+| PostgreSQL Compatible | 14               | Extreme high availability      |
```

Comment:
For the next week or so, Postgres Compatible will not be available.  When it's available it will show `PostgreSQL Compatible | 14 -15`.  I'd say go ahead and post it as 14-15 now and let the Preview catch up.